erica_app/erica/routes.py

```diff
@@ -55,6 +55,22 @@ def send_est(est: EstData, include_elster_responses: bool = False):
         raise HTTPException(status_code=422, detail=e.generate_error_response(include_elster_responses))
 
 
+@app.post(ERICA_VERSION_URL + '/grundsteuer', status_code=status.HTTP_201_CREATED)
+def send_grundsteuer(grundsteuer_data: GrundsteuerData, include_elster_responses: bool = False):
+    """
+    The Grundsteuer data is validated and then send to ELSTER using ERiC. If it is successful, this should return a 200
+    HTTP response with {'transfer_ticket': str, 'pdf': str}. The pdf is base64 encoded binary data of the pdf
+    If there is any error with the validation, this should return a 400 response. If the validation failed with
+    {‘code’ : int,‘message’: str,‘description’: str, ‘validation_problems’ : [{‘code’: int, ‘message’: str}]}
+    or a 400 repsonse for other client errors and a 500 response for server errors with
+    {‘code’ : int,‘message’: str,‘description’: str}
+
+    :param grundsteuer_data: the JSON input data for the land tax declaration
+    :param include_elster_responses: query parameter which indicates whether the ERiC/Server response are returned
+    """
+    return {"request": "successful"}
```

Comment:
I assume that is only for testing purposes and is not already a hint to the actual structure of the response?

Reply:
Yes. testing only. :)